source/NVDAObjects/window/winword.py

```diff
@@ -73,6 +73,27 @@
 wdMaximumNumberOfRows=15
 wdStartOfRangeColumnNumber=16
 wdMaximumNumberOfColumns=18
+
+# Underline style
+wdUnderlineNone = 0

```

Comment:
It is obvious to some, and it was to me in the past, but right now I can't remember what the best way is to find these constants. Could you add a comment at the top of the constants list describing how best to find them.